### ISSUE TITLE
Markdown not getting parsed on page

### ISSUE THREAD

**Issue by rye** [frc461/website-2013#74]
This page (`/pages/About_Us/Team_History`) is not having its markdown properly parsed, it appears.

![screenshot](https://f.cloud.github.com/assets/1566689/2163997/8cdba802-94e6-11e3-823f-47e52e1e400c.png)

Upon some scraping, it appears that this is because the content is inside of a `clearfix`'ed div.

**Comment by braddoeswebdev** [comment]
Yeah, by default Markdown skips anything inside HTML.  There might be an option to turn that off somewhere.

**Comment by rye** [comment]
Could someone with a functioning Rails server that they can test this on check on what effect 3d3ed55bb426ecf4bc48e23f5cd88cf8733120c6 has on it?

**Comment by Sammidysam** [comment]
It makes no change as far as I can tell.

http://s14.postimg.org/8ss090iep/Selection_002.png
http://s14.postimg.org/v5zqvtjch/Selection_003.png

**Comment by cg505** [comment]
Fixed on deploy by removing markdown. Still a bug (or unintentional feature actually).

**Comment by rye** [comment]
Does that mean that this issue is fixed or not (or on only the deploy)?

**Comment by cg505** [comment]
The issue is not at all fixed. The picture demonstrating the problem has
been fixed by removing the markdown from those locations on the deploy.
